TheWarWithin/HunterMarksmanship.lua

```diff
@@ -977,19 +977,24 @@ spec:RegisterAbilities( {
 
         handler = function ()
             removeBuff( "focusing_aim" )
-            removeStack( "precise_shots" )
 
-            if talent.bombardment.enabled then
+            if talent.eagletalons_true_focus.enabled and buff.trueshot.up then
```

Comment:
What makes this necessary?  `removeStack` will remove the buff if you had only 1 stack (or it's defective).  Is a stack of `precise_shots` not removed with every cast the ability?

Reply:
so precise shots stacks to 2 now. During Trueshot, your consuming abilities consume both stacks at once due to eagletalon talent causing those consumers to double-fire. Outside of trueshot, they only consume 1 stack. This change greatly improves stability in the recommendation queue instead of the addon flip-flopping back and forth between an extra multi-shot or not due to reacting to the second buff stack, then reacting again to the second buff stack disappearing.

Reply:
Oops and I just saw a discrepency between my local in-game installation and my PR, aimed shot handler for giving 2 stacks of the buff. Snuck it in.